src/profile.c

```diff
@@ -1005,6 +1005,15 @@ static Int InitKernel (
     return 0;
 }
 
+static Int PostRestore ( StructInitInfo * module )
+{
+    /* When we restore a workspace, we start a new profile. This is the
```

Comment:
I was mildly confused when reading "This" here, until I realized that it referred to the text after the comment.  Perhaps rephrase this as "The OutputtedFilenameList is ..." or "The list of outputted filenames is" ?